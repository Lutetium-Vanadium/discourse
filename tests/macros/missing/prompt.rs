fn main() {
    requestty::questions![Custom { name: "prompt" }];
}
