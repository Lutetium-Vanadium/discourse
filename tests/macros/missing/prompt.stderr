error: missing required option `prompt`
 --> $DIR/prompt.rs:2:34
  |
2 |     requestty::questions![Custom { name: "prompt" }];
  |                                  ^^^^^^^^^^^^^^^^^^
